dev/manualhints

```diff
@@ -34,7 +34,7 @@ How to Proceed with the Reference Manual
   (How to access the two manuals in question:
   The current GAPDoc format version is the one in the development version
   of GAP,
-  which can be compiled by calling `make manuals' in the GAP root directory.
+  which can be compiled by calling `make doc' in the GAP root directory.
```

Comment:
If merged, remember to update https://github.com/gap-system/gap/wiki/Building-main-GAP-manuals as well.